spec/resources/coins/coin_feature_spec.rb

```diff
@@ -127,7 +127,7 @@
     let(:artist) { FactoryBot.create_for_repository(:numismatic_artist) }
 
     before do
-      visit solr_document_path coin
+      visit solr_document_path id: coin.id.to_s
```

Comment:
Why is this change needed?  Wouldn't it route to the same page either way?

Reply:
Oh, yes, I can get rid of this. The problem was that it was only persisting to postgres before, but the change set was being marked as changed as part of creating the record, so the change_set_persister was being fired which would re-save it and it'd end up in solr.